recipes/tests.rb

```diff
@@ -99,6 +99,13 @@
   end
 end
 
+if node['cfncluster']['os'] == 'alinux' || node['cfncluster']['os'] == 'centos7'
+  execute 'check efa_install' do
+    command "ls /opt/amazon/efa || exit 1"
```

Comment:
I think "exit 1" is redundant

Reply:
yep, `ls` doesn't exit with a non-zero exit code if the directory doesn't exist, so I've changed it to use `-d` to check if the directory exists.

Reply:
I've changed this to check for the rpm being installed. `rpm -qa` lists installed rpms. See [1]

```bash
$ rpm -qa | grep libfabric
libfabric-devel-1.7.0amzn1.1-1.amzn2.x86_64
libfabric-1.7.0amzn1.1-1.amzn2.x86_64
$ rpm -qa | grep efa-
efa-0.9.2-1.amzn2.x86_64
```

[1] https://www.cyberciti.biz/howto/question/linux/linux-rpm-cheat-sheet.php